### ISSUE TITLE
The API cannot be used behind a proxy

### ISSUE THREAD

**Issue by jwerba** [jwerba/WhatsAPINet#1]
Im getting error code (407)
"Proxy Authentication Required."

**Comment by jwerba** [comment]
Tell .NET application to use default proxy configuration

``` C#
static WhatsRegisterV2()
{
            System.Net.WebRequest.DefaultWebProxy.Credentials = System.Net.CredentialCache.DefaultNetworkCredentials;
}
```

The default behaviour of WebRequest.DefaultWebProxy is to use the same underlying settings as used by Internet Explorer.

**Comment by jwerba** [comment]
Solved by commit [f0cc5d791020fc06d75eb7ed69fbe628d61cd378](https://github.com/jwerba/WhatsAPINet/commit/f0cc5d791020fc06d75eb7ed69fbe628d61cd378)
